store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -351,17 +384,23 @@ protected boolean isColumnQualifierBytesValid(final byte[] bytes, final int numP
 
     @Override
     public long buildTimestamp(final String group, final Properties properties) {
-        Long timestamp = null;
-        if (null != timestampProperty) {
-            timestamp = (Long) properties.get(timestampProperty);
+        Long timestamp;
+
+        // Allow override via property
+        if (timestampProperty != null) {
+            return (Long) properties.get(timestampProperty);
```

Comment:
I didn't really see much point in assigning to the variable as this way it will be left uninitialised too saving memory, a variable doesn't have to be used on all possible branches, I'd argue this is just as clear to follow.